srv/salt/ceph/tests/restart/rgw/same_pid.sls

```diff
@@ -1,7 +1,7 @@
 
 Check pid:
   cmd.run:
-    - name: "[ `pgrep radosgw` ==  `cat /tmp/restart.pid` ]"
+    - name: "test \"$(pgrep radosgw)\" = \"$(cat /tmp/restart.pid)\""
```

Comment:
Another possibility would be to use the bash built-in `[[ . . . ]]` which doesn't require the variables to be in double-quotes.

Remember: running shell commands via `cmd.run` is fun!